vizro-core/examples/kpi/app.py

```diff
@@ -0,0 +1,271 @@
+"""Example to show dashboard configuration."""
+
+import pandas as pd
+import vizro.models as vm
+from utils._charts import COLUMN_DEFS, KPI, bar, chloropleth, line, pie
+from utils._helper import clean_data_and_add_columns
+from vizro import Vizro
+from vizro.actions import filter_interaction
+from vizro.tables import dash_ag_grid
+
+# DATA --------------------------------------------------------------------------------------------
+df_complaints = pd.read_csv("data/Financial Consumer Complaints.csv")
+df_complaints = clean_data_and_add_columns(df_complaints)
+
+vm.Page.add_type("components", KPI)
+vm.Container.add_type("components", KPI)
+
+# TODO: Enable selection of year filter
+# TODO: Enable current year vs. past year comparison
+# TODO: Enable dynamic KPI Cards
+# TODO: Make KPI Card row horizontally scrollable if no space
+# TODO: Bar - How to enable drill-downs for Issue/Sub-issue and Product/Sub-product?
+# TODO: Bar - Reformat numbers with commas in bar chart
+# TODO: Bar - Left-align y-axis labels
+# TODO: Bar - Shorten labels
+# TODO: Line - Customize function to always show selected year vs. past year
+
+
+page_exec = vm.Page(
+    title="Executive View",
+    layout=vm.Layout(
+        grid=[
+            [0, 1, 2, 3, 4, 5],
+            [0, 1, 2, 3, 4, 5],
+            [6, 6, 6, 7, 7, 7],
+            [6, 6, 6, 7, 7, 7],
+            [6, 6, 6, 7, 7, 7],
+            [6, 6, 6, 7, 7, 7],
+            [6, 6, 6, 8, 8, 8],
+            [6, 6, 6, 8, 8, 8],
+            [6, 6, 6, 8, 8, 8],
+            [6, 6, 6, 8, 8, 8],
+        ],
+    ),
+    components=[
+        # Note: For some KPIs the icon/sign go in opposite directions as an increase e.g. in complaints is negative
```

Comment:
This is a good point. Will we need something in our KPI card function to indicate "positive is good/bad"? I've definitely seen similar arguments in other things though can't remember where now.

Reply:
We had the discussion about colors no :)

Reply:
This is a different discussion. :) We previously discussed whether blue and red are considered good or bad, and we've aligned that in our demo examples; we consider blue as good and red as bad. Nothing has changed here.

What I am referring to here is that the `delta` is not always perceived the same, but it depends on the KPI. For example, we can't encode "positive delta is always good" in the KPI cards because a positive delta can be "bad" if the KPI is the number of complaints for example, as we want fewer complaints. So that's what I am referring to here - we need an additional argument in the KPI Card where people can provide "positive delta is bad" or "positive delta is good."